### PR TITLE
Porting to raspberry pi2.

### PR DESCRIPTION
**Pull request by Adenilson**
Please consider merging this as it was required to compile servo on raspberry pi.

### CODE REVIEW

File: makefile.cargo [hunk 1]
```diff
@@ -47,7 +47,9 @@ CXXFLAGS += \
 	-Isrc/utils \
 	-Ithird_party/etc1 \
 	-Ithird_party/ktx \
-	-iquote src/gpu
+	-iquote src/gpu \
+	-I/usr/include/freetype2
```
Comment (pcwalton) [inline makefile.cargo]: Will this cause problems on e.g. Mac? I have no `/usr/include/freetype2` there.

**Comment by Adenilson** [comment]
let me try it out.

**Comment by Adenilson** [comment]
As I expected, the compiler will just ignore if a folder doesn't exist. 

Quick test: 
1500002299-ACAV:temp a.cavalcanti$ gcc -Wall -I/foo/bar foo.c 
1500002299-ACAV:temp a.cavalcanti$ 

Or for the longer version: https://gist.github.com/Adenilson/7cf93b9ec93632f2cb04

**Comment by pcwalton** [comment]
I guess it's fine then.

**Pull request merged**